int64 trigNum
int64 sysTime
float32 stateErr
